test/test_case_test.dart

```diff
@@ -22,5 +23,11 @@ void main() {
       final result = await TestCase.getAll(1);
       expect(result.asJson, sampleForTestCases);
     });
+
+    test('#getHistory', () async {
```

Comment:
This should be in its own test file (`test/test_case_history_test.dart`) and be annotated `.get` instead